CHANGELIST

```diff
@@ -1,8 +1,8 @@
 2.1.0  [overall]   documentation updates, small fixes (@HavardLine, @croadfeldt, @jimsalterjrs, @jim-perkins, @kr4z33, @phreaker0)
        [syncoid]   do not require user to be specified for syncoid (@aerusso)
        [syncoid]   implemented option for keeping sync snaps (@phreaker0)
-       [syncoid]   use sudo if neccessary for checking pool capabilities regarding resumeable send (@phreaker0)
-       [syncoid]   catch another case were the resume state isn't availabe anymore (@phreaker0)
+       [syncoid]   use sudo if necessary for checking pool capabilities regarding resumable send (@phreaker0)
+       [syncoid]   catch another case were the resume state isn't available anymore (@phreaker0)
```

Comment:
Some projects don't like changing changelogs. Happy to drop this or anything else.